src/Tizen.Uix.Tts/Tizen.Uix.Tts/TtsClient.cs

```diff
@@ -203,6 +203,37 @@ public enum Voice
         Child
     };
 
+    /// <summary>
+    /// Enumeration for the states of TTS service.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    public enum ServiceState
+    {
+        /// <summary>
+        /// Ready state.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
```

Comment:
you don't need the \<since_tizen\> in enum field if the field was introduced in the initial version of the enum - i.e. if the version here is the same as the \<since_tizen\> of the containing enum

Reply:
Thank you for checking. I will remove it soon.